ingest.py

```diff
@@ -3,7 +3,7 @@
 from typing import List
 from dotenv import load_dotenv
 
-from langchain.document_loaders import TextLoader, PDFMinerLoader, CSVLoader
+from langchain.document_loaders import TextLoader, PDFMinerLoader, CSVLoader, UnstructuredEmailLoader, OutlookMessageLoader
```

Comment:
After reviewing LangChain documentation for the two loaders, I think you are missing new requirements to be installed, that should be added to the requirements file:

unstructured
extract_msg

Reply:
Oh sorry. Added the two dependencies to the `requirements.txt`.